src/microbe_stage/CompoundBag.cs

```diff
@@ -130,6 +130,11 @@ public bool IsSpecificallySetUseful(Compound compound)
         return usefulCompounds.Contains(compound);
     }
 
+    public bool AreAnySpecificallySetUseful(List<Compound> compounds)
+    {
+        return usefulCompounds.Intersect(compounds).Any();
```

Comment:
I wonder if this is more or less efficient than the way I'd write this:
```c#
return compounds.Any(c => usefulCompounds.Contains(c));
```

Reply:
No idea. I can change it if you'd prefer.

Reply:
I'd prefer it so that code is more uniform. If it turns out Intersect is actually faster we have a bunch of code to migrate anyway.

Reply:
I'll commit changing this to `return compounds.Any(usefulCompounds.Contains);`